Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -773,6 +773,7 @@ def create_user_iam(default_base_dn, args, mapper_out, disabled_users_group_cn):
         else:
             user_dn = generate_dn_and_remove_from_user_profile(ad_user)
             object_classes = ["top", "person", "organizationalPerson", "user"]
+            ad_user.pop('ou')
```

Comment:
how can you be sure there is more than one?
what happens if there is only one/none?
```suggestion
            ad_user.pop('ou', None)
```

Reply:
So In `ad_user` we already have the `ou` field which is populated to the `user_dn`. So we removed it from the `ad_user` when creating the user in AD.
There will always be one 'ou'. Its mandatory and exists in the mapper. it will fail the creation if there is None